pkg/trace/obfuscate/sql.go

```diff
@@ -326,18 +320,15 @@ func (o *Obfuscator) obfuscateSQL(span *pb.Span) {
 			span.Meta[sqlQueryTag] = span.Resource
 		}
 		span.Resource = nonParsableResource
-		tags = append(tags, "outcome:error")
 		return
 	}
 
-	tags = append(tags, "outcome:success")
 	span.Resource = oq.Query
 
 	if len(oq.TablesCSV) > 0 {
 		traceutil.SetMeta(span, "sql.tables", oq.TablesCSV)
 	}
 	if span.Meta != nil && span.Meta[sqlQueryTag] != "" {
-		// "sql.query" tag already set by user, do not change it.
```

Comment:
Why did you remove this? We need this comment. When the `sql.query` tag is set by the user we don't override it.

Reply:
Sorry, I misread this comment.
I will put it back.